### ISSUE TITLE
VideoMediaReceived does not get called when AudioVideoPlaybackBot joins a meeting

### ISSUE THREAD

**Issue by pavanjains** [microsoftgraph/microsoft-graph-comms-samples#164]
I am using the latest version of AudioVideoPlaybackBot  to join a meeting. In my case I do not see the VideoMediaReceived  getting called. I looked into the code and CallHandler() constructor has this line

this.Call.GetLocalMediaSession().VideoSockets.FirstOrDefault().VideoMediaReceived += this.OnVideoMediaReceived;

However video on my case comes on socket #3, so shouldn't the code be 

           foreach (var videoSocket in this.Call.GetLocalMediaSession().VideoSockets)
            {
                videoSocket.VideoMediaReceived += this.OnVideoMediaReceived;
            }

Please let me know if my understanding is correct or not.

**Comment by nikskhubani** [comment]
Yes I believe it should be the test case. For me the code is something like this


```

                this.videoSockets = this.mediaSession.VideoSockets?.ToList();
                foreach (var videoSocket in this.videoSockets)
                {
                    videoSocket.VideoMediaReceived += this.OnVideoMediaReceived;
                }
```
